serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_quarkus_cli.adoc

```diff
@@ -0,0 +1,84 @@
+// links
+:quarkus_k8s_deploy_url: https://github.com/quarkusio/quarkus/issues/26385
+
+[[proc-deploy-sw-application-quarkus-cli]]
+=== Deploying your workflow application using Quarkus CLI
+
+.Prerequisites
+* Workflow application is installed.
+* Quarkus CLI is installed. +
+For more information, see link:{quarkus_cli_url}[Building Quarkus Apps with Quarkus command line interface (CLI)].
+* You have completed the steps in link:#proc-deploy-sw-application-kubectl[Deploying your workflow application using `kubectl`]
+
+.Procedure
+. Add the Quarkus extensions to deploy the `knative` service
++
+--
+You can add the Kubernetes and the Kogito Knative extensions to your project with the Quarkus CLI:
+
+.Add kubernetes and Kogito knative extensions to the project with Quarkus CLI
+[source,shell]
+----
+quarkus extension add kubernetes
+quarkus extension add kogito-addons-quarkus-knative-eventing
+----
+--
+. To deploy your workflow application using Quarkus CLI, set the following system properties in `application.properties` file:
++
+--
+.Required system properties
+[source,properties]
+----
+quarkus.knative.name=greeting-quarkus-cli <1>
+quarkus.kubernetes.deployment-target=knative <2>
+quarkus.kubernetes.deploy=true <3>
+quarkus.container-image.push=false <4>
+----
+
+<1> Property for the Knative resource name
+<2> Property to set whether or not deploy on an active Kubernetes cluster
+<3> {container_push}
+
+[IMPORTANT]
+====
+This feature works with Quarkus 2.10.2.Final or later. For more information, see
+link:{quarkus_k8s_deploy_url}[link].
+====
+--
+
+. Verify if the workflow application is deployed correctly:
```

Comment:
How do you deploy the application ? Which Maven/Gradle command ?

Reply:
it should be quarkus-cli, will add it.